core/templates/pages/exploration-editor-page/exploration-editor-page.component.html

```diff
@@ -49,8 +49,8 @@ <h3 class="e2e-test-joyride-title">Preview</h3>
            [ngbTooltip]="'Translations'"
            placement="bottom"
            tabindex="0"
-           aria-label="Exploration Translation Button"
-           (keydown.enter)="!connectedToInternet || editabilityService.isLockedByAdmin() || selectTranslationTab()">
```

Comment:
Hi @nicoleSosa, would you please tell me why was `(keydown.enter)="!connectedToInternet || editabilityService.isLockedByAdmin() || selectTranslationTab()` removed? Thanks!